src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -217,6 +217,12 @@ public boolean isExternalSubsParsed() {
 	private byte[] h264_annexB;
 
 	/**
+	 * @deprecated Use standard getter and setter to access this variable.
+	 */
+	@Deprecated
+	public String truncated;
+
+	/**
 	 * Not stored in database.
```

Comment:
OK, fine. Yes, i'll correct that.
Thanks for reviewing. Yes, it is for me, as far as only my code is concerned :P

I checked at your link, in the only thing that i understood and remember is that the object language was created by norvegian people around 1960 and developped by an american 10 years later...
It's amazing how your mother language can look like chineese to you when it's technical term that don't make sens to you, also if you understand all the words, put together no meaning come...
At my skill level, a book like Java for the nuls will be more appropriate :wink: , if one day i really decide to learn _Java_ more than 3 lines of code...